sdk/loadtesting/load-testing-rest/src/outputModels.ts

```diff
@@ -491,42 +380,57 @@ export interface DimensionValueOutput {
   value?: string;
 }
 
-/** Metrics dimension values. */
 export interface DimensionValueListOutput {
-  /** The dimension values */
-  value?: Array<string>;
-  /** Link for the next set of values in case of paginated results, if applicable */
-  nextLink?: string;
+  value: Record<string, any>;
```

Comment:
breaking here, @MaryGao need to follows up why this happens?

Reply:
Fixed in the branch as well